bundles/org.openhab.automation.pythonscripting/src/main/java/org/openhab/automation/pythonscripting/internal/PythonScriptEngineFactory.java

```diff
@@ -132,11 +132,13 @@ public void scopeValues(ScriptEngine scriptEngine, Map<String, Object> scopeValu
 
     private void initHelperLib() {
         try {
+            String resourceSeparator = "/";
```

Comment:
I think this could be hardcoded since it will be `/` regardless of operating system? Alternatively, you could make it a constant, since its value will never change.

Reply:
is fixed